### PR TITLE
OGD-563 Individuals Income P1.0 - SA Trusts Resource - SANDBOX + DOCU…

### PR DESCRIPTION
**Pull request by edgarjimenez**
…MENTATION

### CODE REVIEW

File: app/uk/gov/hmrc/individualsincomeapi/domain/SaSubmission.scala [hunk 1]
```diff
@@ -71,4 +71,16 @@ object SaTaxReturnSummaries {
       TaxYear.fromEndYear(desSAIncome.taxYear.toInt),
       desSAIncome.returnList.map(sa => SaTaxReturnSummary(sa.utr, sa.incomeFromSelfAssessment.getOrElse(0.0))))
   }
+}
+
+case class SaAnnualTrusts(taxYear: TaxYear, trusts: Seq[SaAnnualTrust])
+case class SaAnnualTrust(utr: SaUtr, trustIncome: Double)
```
Comment (tom59) [inline app/uk/gov/hmrc/individualsincomeapi/domain/SaSubmission.scala]: can we follow the same pattern than the others?
SaAnnualTrusts and SaTrustsIncome

Comment (edgarjimenez) [reply app/uk/gov/hmrc/individualsincomeapi/domain/SaSubmission.scala]: 👍

File: app/uk/gov/hmrc/individualsincomeapi/domain/JsonFormatters.scala [hunk 2]
```diff
@@ -73,6 +73,9 @@ object JsonFormatters {
 
   implicit val formatSaTaxReturnSummary = Json.format[SaTaxReturnSummary]
   implicit val formatSaTaxReturnSummaries = Json.format[SaTaxReturnSummaries]
+
+  implicit val formatSaAnnualTrust = Json.format[SaAnnualTrustIncome]
```
Comment (tom59) [inline app/uk/gov/hmrc/individualsincomeapi/domain/JsonFormatters.scala]: very small change, can it be renamed?

Comment (edgarjimenez) [reply app/uk/gov/hmrc/individualsincomeapi/domain/JsonFormatters.scala]: 👍

**Pull request merged**